src/EditorFeatures/Core/LanguageServer/AlwaysActiveLanguageClientEventListener.cs

```diff
@@ -42,6 +42,19 @@ public void StartListening(Workspace workspace)
     {
         // Trigger a fire and forget request to the VS LSP client to load our ILanguageClient.
         _ = LoadAsync();
```

Comment:
Anecdotally under the debugger on my machine, this line is not needed as the `SolutionAdded` event fires after this anyway, but I wasn't sure if that was guaranteed so I left it. There is no real downside to calling `LoadAsync` multiple times anyway, other than an extra yield or two.

If anyone on Roslyn wants to guarantee that `SolutionAdded` will always happen after `StartListening` then I'll gladly get rid of this :)